saleor/graphql/middleware.py

```diff
@@ -58,6 +59,51 @@ def app_middleware(next, root, info, **kwargs):
     return next(root, info, **kwargs)
 
 
+class ReadOnlyMiddleware:
+    ALLOWED_MUTATIONS = [
```

Comment:
I noticed we don't allow users to register new accounts. Is it meant?

Reply:
Yes it is, we don't want them to create accounts on demo.